plugins/modules/zabbix_script.py

```diff
@@ -234,9 +234,11 @@ def generate_script_config(self, name, script_type, command, scope, execute_on,
                 '',
                 'manual_event_action'], scope)),
             'groupid': groupid,
```

Comment:
Remove comma?
Remove the next empty line?

Reply:
Comma removed. I think empty line (you mean one after the dict?) improves readability. Looks fine now? Thanks.

Reply:
Looks ok to me, thanks! But I am not a reviewer, just noted some "not very clean" code. Let's wait for maintainers.